config/application.rb

```diff
@@ -1,25 +1,40 @@
-require File.expand_path('../../env', __FILE__)
-require File.expand_path('../boot', __FILE__)
+require_relative '../env'
+require_relative 'boot'
 
+require "rails"
+# Pick the frameworks you want:
+require "active_model/railtie"
+require "active_job/railtie"
+# require "active_record/railtie"
+# require "active_storage/engine"
 require "action_controller/railtie"
-require "active_resource/railtie"
+# require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
+require "action_view/railtie"
+# require "action_cable/engine"
 require "sprockets/railtie"
+require "rails/test_unit/railtie"
 
-if defined?(Bundler)
-  # If you precompile assets before deploying to production, use this line
-  Bundler.require(*Rails.groups)
-  # If you want your assets lazily compiled in production, use this line
-  # Bundler.require(:default, :assets, Rails.env)
-end
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
 
 module PhasersSample
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 6.0
 
+    config.encoding = "utf-8"
     config.autoload_paths += %W(#{config.root}/lib)
 
-    config.encoding = "utf-8"
-    config.filter_parameters += [:password]
-    config.assets.enabled = true
-    config.assets.version = '1.0'
+    if ENV['APP_HOST']
+      config.hosts << ENV['APP_HOST']
```

Comment:
Optionally append a allowed host via ENV variable, which would allow someone in ops to deploy the app to whatever domain they need to (qa envs, staging, production, local, etc).